syntax = "proto3";

package chat;

message Room {
  string room_id = 1;
  string name = 2;
  repeated string user_ids = 3;
}
